packages/widget-chat/src/containers/message-composer/index.js

```diff
@@ -62,14 +65,14 @@ class MessageComposer extends Component {
 MessageComposer.propTypes = {
   placeholder: PropTypes.string,
   setMessage: PropTypes.func.isRequired,
-  submitMessage: PropTypes.func.isRequired,
-  value: PropTypes.string
```

Comment:
Is it? I didn't see it being used in the main container. When would be use passing a value into the message? During message editing of a posted message? I don't think we have that ability.